### PR TITLE
INT-3753: Add Channel Send Count to IMBE

### PR DESCRIPTION
**Pull request by garyrussell**
JIRA: https://jira.spring.io/browse/INT-3753

Missing accessor for channel send counts.

**cherry-pick to 4.1.x**

### CODE REVIEW

File: spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java [hunk 1]
```diff
@@ -630,6 +630,18 @@ public long getSourceMessageCountLong(String name) {
 		return -1;
 	}
 
+	public int getChannelSendCount(String name) {
+		return (int) getChannelSendCountLong(name);
+	}
+
+	public long getChannelSendCountLong(String name) {
+		if (channelsByName.containsKey(name)) {
+			return channelsByName.get(name).getSendCountLong();
```
Comment (artembilan) [inline spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java]: So, `getSendErrorCount()` will we cover on demand?
Wouldn't it be better just expose `getChannelMetrics(String name)` ?

Comment (garyrussell) [inline spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java]: I will add the error count and also expose metrics objects.

**Comment by garyrussell** [comment]
Rebased, squashed, pushed.

When cherry-picking, fix minor conflicts in test case and remove

    		assertSame(intermediate, messageChannelsMonitor.getChannelMetrics(intermediateChannelName));

**Comment by artembilan** [comment]
Merging...

**Comment by artembilan** [comment]
... and cherry-picked with mentioned conflict fix

**Pull request merged**